### ISSUE TITLE
Installing a clock does not work with a Date object

### ISSUE THREAD

**Issue by amsul** [sinonjs/lolex#73]
As mentioned in the docs: https://github.com/sinonjs/lolex#var-clock--lolexcreateclocknow-looplimit

> The now argument may be a number (in milliseconds) or a Date object.

However, this does not work for a Date object.

It seems as though this line (https://github.com/sinonjs/lolex/blob/f464603ef78b585aabb1ad461b26a3267989536c/src/lolex-src.js#L620) needs to be modified to:

``` js
if (typeof target === 'number' || target instanceof Date) {
  // ...
}
```

**Comment by julen** [comment]
As a temporal workaround one can pass a falsy value to `lolex.install` and provide a `Date` object in the second argument: `lolex.install(undefined, new Date('2016-07-08T11:20:11+00:00'));`

**Comment by fatso83** [comment]
@amsul: seeing that you found the relevant change to be made, would you care to make a PR? Please add tests that shows the feature working as well, if you do.

**Comment by fatso83** [comment]
You might want to follow PR #89. It needs some research help in getting further.

**Comment by fatso83** [comment]
This was fixed in 5bd7dff2bcd66db97bbc6b36a20a9ddb4f411b23
